### PR TITLE
LADX: THIEF

### PR DESCRIPTION
**Pull request by zig-for**
## What is this fixing or adding?
Makes sure everyone knows when you've been a THIEF (only activates April 1)

## How was this tested?
Stole item. Client sent itself the proper alias command, would not let me unalias unless I nuked my save file.

## If this makes graphical changes, please attach screenshots.
image

### CODE REVIEW

File: LinksAwakeningClient.py [hunk 1]
```diff
@@ -404,6 +410,10 @@ async def main_tick(self, item_get_cb, win_cb, deathlink_cb):
         if await self.is_victory():
             await win_cb()
 
+        today = datetime.date.today()
```
Comment (Berserker66) [inline LinksAwakeningClient.py:413]: I believe getting the current date on every tick is considered expensive.

Comment (zig-for) [reply LinksAwakeningClient.py:413]: It shouldn't be _that_ expensive - but you're right, I can check on open or every hour or something.

**Comment by zig-for** [comment]
Open to closing it but it’s an April Fools joke

**Comment by ThePhar** [comment]
I think this should probably be closed as well. Changing (and preventing revert) of alias even on a single day seems like something some players could complain or be confused about.

**Pull request closed without merging**